GUI/SearchTaskWindow.xaml

```diff
@@ -48,15 +48,36 @@
                         <StackPanel>
                             <GroupBox Header="Deisotoping">
                                 <StackPanel>
-                                    <CheckBox x:Name="useProvidedPrecursor" Content="Use Provided Precursor" IsChecked="True" />
-                                    <CheckBox x:Name="deconvolutePrecursors" Content="Deconvolute Precursors" IsChecked="True" />
+                                    <CheckBox x:Name="useProvidedPrecursor" Content="Use Provided Precursor" IsChecked="True" ToolTipService.ShowDuration="999999" ToolTipService.InitialShowDelay="1000">
+                                        <CheckBox.ToolTip>
+
+                                            <TextBlock>
+                                                Use the charge states and precursor masses determined by the mass spectrometer.
```

Comment:
Search for precursor masses determined by the instrument control

Reply:
controller?

Reply:
add "as reported in the raw data file".